runtimes/go/appruntime/apisdk/cors/cors.go

```diff
@@ -67,8 +67,20 @@ func Options(cfg *config.CORS, staticAllowedHeaders, staticExposedHeaders []stri
 		AllowOriginRequestFunc: func(r *http.Request, origin string) bool {
 			// If the request has credentials, look up origins in AllowOriginsWithCredentials.
 			// Credentials are cookies, authorization headers, or TLS client certificates.
-			hasCreds := len(r.Cookies()) > 0 || r.Header["Authorization"] != nil || (r.TLS != nil && len(r.TLS.PeerCertificates) > 0)
-			if hasCreds {
+			hasCreds := func(r *http.Request) bool {
+				if len(r.Cookies()) > 0 || r.Header["Authorization"] != nil || (r.TLS != nil && len(r.TLS.PeerCertificates) > 0) {
```

Comment:
```suggestion
				if len(r.Cookies()) > 0 || len(r.Header["Authorization"]) > 0 || (r.TLS != nil && len(r.TLS.PeerCertificates) > 0) {
```

Reply:
changed to using `.Values()`, as accessing the map directly isnt case-sensitive afaict